src/runtime_src/core/pcie/tools/xbmgmt/cmd_config.cpp

```diff
@@ -177,6 +178,17 @@ static void showDevConf(std::shared_ptr<pcidev::pci_device>& dev)
         std::cout << "\t" << "Runtime clock scaling enabled status: " <<
             lvl << std::endl;
     }
+
+    errmsg = "";
+    dev->sysfs_get("xmc", "scaling_threshold_power_override", errmsg, svl);
+    if (!errmsg.empty()) {
+        std::cout << "can't read scaling_threshold_power_override from " <<
```

Comment:
Is this an error, warning/caution or info? can you print the tag in the beginning: "CAUTION: Can't read ...."
Please make the change in the code above this as well